configurations/default/env.yml.tmp

```diff
@@ -15,6 +15,7 @@ AUTH0_API_CLIENT: your-api-client-id
 AUTH0_API_SECRET: your-api-secret-id
 AWS_PROFILE: default
 DEFAULT_USAGE_PLAN_ID: usage-plan-id
+OTP_MIDDLEWARE_URI_ROOT: http://otp-middleware-server:4567
```

Comment:
This only appears in a commented code block. Do we still need this?

Reply:
It is only being potentially used in test, so maybe we can remove from this file and keep what you have in `test/env.yml`.

Reply:
Removed. Testing to be addressed.